wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/WEmailFieldWebElementTest.java

```diff
@@ -31,7 +30,7 @@ public WEmailFieldWebElementTest() {
 	public void testFeaturesOfEmailField() {
```

Comment:
This test is testing validation messages - which is fine - but not "features of email field" other than the diagnostics. Maybe change this text method name or actually test a few more of the "features" of WEmailField.